controller/productController.js

```diff
@@ -0,0 +1,83 @@
+const productService = require('../services/productService')
+const middleJwt = require('../middleware/jwt');
+
+const productAll = async(req,res) =>{
+    try{
+        const {brandName,scentName} = req.query
+        const products = await productService.getProducts(brandName,scentName)
+        console.log(products[0].scent_desc)
+        res.status(200).json(products)
```

Comment:
```suggestion
        res.status(200).json({ products })
```

데이터 그냥 배열로 보내시면 안되고 객체 형태로 보내셔야 합니다. key-value 형태로 변경해 주세요!